src/components/chart/common-chart-options.js

```diff
@@ -57,6 +57,9 @@ class CommonChartOptions {
             },
             accessibility: {
                 enabled: true,
+                highContrastTheme: {
+                    colors: ['#a6f0ff', '#70d49e', '#e898a5', '#007faa', '#f9db72', '#f45b5b', '#1e824c', '#e7934c', '#dadfe1', '#a0618b'],
+                },
```

Comment:
Do these need to be set here when they are already set in the css?

Reply:
It was originally set for the area chart only, but I’ve now updated it to apply the styles via CSS as well.